src/api/app/controllers/source_controller.rb

```diff
@@ -1560,11 +1560,11 @@ def package_command_linktobranch
 
     # convert link to branch
     rev = ''
-    if not pkg_rev.nil? and not pkg_rev.empty?
+    if !pkg_rev.nil? && !pkg_rev.empty?
       rev = "&orev=#{pkg_rev}"
     end
     linkrev = ''
-    if not pkg_linkrev.nil? and not pkg_linkrev.empty?
+    if !pkg_linkrev.nil? && !pkg_linkrev.empty?
```

Comment:
This wouldn't be exactly the same

If `pkg` is `" "` 

  `!pkg_rev.nil? && !pkg_rev.empty?` is true
  `pkg_rev.present?` is false